spark/v3.4/spark-runtime/src/integration/java/org/apache/iceberg/spark/SmokeTest.java

```diff
@@ -169,9 +169,9 @@ public void testCreateTable() {
 
   @Test
   public void showView() {
-    sql("DROP VIEW IF EXISTS %s", "test");
```

Comment:
this test started to fail only in Spark 3.4 when backporting the view changes to `SparkSessionCatalog`. There is some funkiness going on with the `spark_catalog` in Spark 3.4 tests where the `spark_catalog` is reusing some settings from a previous test run. I'll investigate that separately and for now decided to move the `DROP` to the end of the test (and aligned the same thing for the Spark 3.5 `SmokeTest`.

The TLDR of this test failure is: 
* a previous catalog config created a view named `test`
* the next test configuration tries to drop this view where the catalog type is `hive`
* Hive fails when trying to load the view in order to see if it exists [here](https://github.com/apache/iceberg/blob/6e9e07aa0f35197bb23b218d23716e928d1c2814/hive-metastore/src/main/java/org/apache/iceberg/hive/HiveViewOperations.java#L96) due to the view not being an iceberg view